src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double base;
+    private final double lateralSide;
+
+    IsoscelesTrapezoid(String color, double base, double lateralSide) {
+        super(color);
+
+        this.base = base;
+        this.lateralSide = lateralSide;
+    }
+
+    @Override
+    public double getArea() {
+        double firstLeg = base / 2;
+        double height = Math.pow(lateralSide, 2) - Math.pow(firstLeg, 2);
```

Comment:
Think about which variables should be local in the method and which should be class-level. The `firstLeg` seems to be a property of the trapezoid that could be reused in other methods and should be a field of the class.